src/utils/index.test.js

```diff
@@ -143,7 +143,7 @@ it('sortRunsByStartTime', () => {
     sort is stable on all modern browsers so
     input order is preserved for b and e
    */
-  const sortedRuns = [b, e, f, c, d, a];
+  const sortedRuns = [e, b, f, c, d, a];
```

Comment:
I don't see any change to the implementation so not sure why this change is required. This test should be failing as `b` and `e` are expected to remain in that order, see comment

Reply:
I had only changed this as it was failing locally for me and the order of them was changing when I printed out the expected, I will change this back now 😅